ntopng-listing/functions.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * Copyright 2005 - 2022 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+/**
+ * @param array{url: string, login: string, password: string, base_url: string, uri: string} $preferences
+ * @return string|bool
+ * @throws Exception
+ */
+function callProbe(array $preferences): string|bool
```

Comment:
```suggestion
function callProbe(array $preferences): string
```
As CURLOPT_RETURNTRANSFER option is set, it will return the result on success and false would throw an exception